projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { PluginPrePublishPanel, PluginPostPublishPanel } from '@wordpress/edit-post';
+import { createInterpolateElement, useEffect, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getSubscriberCount } from './api';
+import InspectorNotice from '../../shared/components/inspector-notice';
+import './panel.scss';
+
+export default function SubscribePanels() {
+	const [ subscriberCount, setSubscriberCount ] = useState( null );
+	useEffect( () => {
+		getSubscriberCount( count => setSubscriberCount( count ) );
+	}, [] );
+
+	return ! Number.isFinite( subscriberCount ) || subscriberCount <= 0 ? null : (
+		<>
+			<PluginPrePublishPanel
+				className="jetpack-subscribe-pre-publish-panel"
+				initialOpen
+				title={ __( 'Subscribers', 'jetpack' ) }
+			>
+				<InspectorNotice>
+					{ createInterpolateElement(
+						sprintf(
+							/* translators: %s is the number of subscribers */
+							_n(
+								'This post will be sent to <span>%s reader</span>',
+								'This post will be sent to <span>%s readers</span>',
```

Comment:
Should we add periods to those sentences?

```suggestion
								'This post will be sent to <span>%s reader</span>.',
								'This post will be sent to <span>%s readers</span>.',
```

Reply:
I'd had those periods in my initial implementation, but the design specs called for an omission of periods on closer inspection.